content/news/052/index.md

```diff
@@ -65,7 +65,16 @@ If needed, a section can be split into subsections with a "------" delimiter.
 
 ## Announcements
 
-TODO: I'll be working on adding an email subscription this weekend, so I'll write about it here once that's done :) - Jan
+Since years, our readers have asked for a way to subscribe to this newsletter by email. This again came up in our [recent survey][survey].
```

Comment:
```suggestion
For years, our readers have asked for a way to subscribe to this newsletter by email. This again came up in our [recent survey][survey].
```

not 100% sure about this, but I've been taught that `since` is used exclusively with fixed (absolute) dates

Reply:
You're right!